src/Files.App/ViewModels/SettingsViewModels/FoldersViewModel.cs

```diff
@@ -239,6 +244,60 @@ public bool CalculateFolderSizes
 			}
 		}
 
+		private int selectedDefaultSortingIndex;
+		public int SelectedDefaultSortingIndex
+		{
+			get => selectedDefaultSortingIndex;
+			set
+			{
+				if (SetProperty(ref selectedDefaultSortingIndex, value))
+				{
+					OnPropertyChanged(nameof(SelectedDefaultSortingIndex));
+					DefaultSortOption = value == FileTagSortingIndex ? SortOption.FileTag : (SortOption)value;
+				}
+			}
+		}
+
+		private int selectedDefaultGroupingIndex;
+		public int SelectedDefaultGroupingIndex
+		{
+			get => selectedDefaultGroupingIndex;
+			set
+			{
+				if (SetProperty(ref selectedDefaultGroupingIndex, value))
+				{
+					OnPropertyChanged(nameof(SelectedDefaultGroupingIndex));
+					DefaultGroupOption = value == FileTagGroupingIndex ? GroupOption.FileTag : (GroupOption)value;
```

Comment:
```suggestion
					FoldersSettingsService.DefaultDirectorySortOption = value == FileTagGroupingIndex ? GroupOption.FileTag : (GroupOption)value;
```
```suggestion
					DefaultGroupOption = value == FileTagGroupingIndex ? GroupOption.FileTag : (GroupOption)value;
```
If we do this, we can remove the `DefaultSortOption` property from line 275.

Reply:
You probably meant FoldersSettingsService.DefaultGroupOption

Reply:
Yeah

Reply:
Can you do the same for Sorting?

Reply:
Yes